File: unimplemented_commands/cat.js
// Description
//   Moss cat command
//   Why that wasn't there already I don't even know
//
// Commands:
//   `!cat` - Shows the moss cat, brings torture to 2310 students

module.exports = function (robot) {
	robot.respond(/!?cat/i, function (res) {
		res.send("```\n" +
                "         __..--''``\\--....___   _..,_\n" +
                "     _.-'    .-/\";  `        ``<._  ``-+'~=.\n" +
                " _.-' _..--.'_    \\                    `(^) )\n" +
                "((..-'    (< _     ;_..__               ; `'   fL\n" +
                "           `-._,_)'      ``--...____..-'\n```");
  });
};

File: uqcsbot/scripts/cat.py
from uqcsbot import bot, Command


@bot.on_command("cat")
def handle_cat(command: Command):
    """
    `!cat` - Displays the moss cat. Brings torture to CSSE2310 students.
    """
    cat = "\n".join(("```",
                     "         __..--''``\\--....___   _..,_            ",
                     "     _.-'    .-/\";  `        ``<._  ``-+'~=.     ",
                     " _.-' _..--.'_    \\                    `(^) )    ",
                     "((..-'    (< _     ;_..__               ; `'   fL",
                     "           `-._,_)'      ``--...____..-'         ```"))
    bot.post_message(command.channel_id, cat)
